inc/planning.class.php

```diff
@@ -2480,13 +2480,13 @@ static function returnNewEventByType ($event, $content, $tooltip, $begin, $end,
       switch ($viewname) {
          case 'resourceTimeGridDay' :
             if ($event['itemtype'] == TicketTask::class) {
-                  $title = _n('Ticket task', 'Ticket tasks', $nb);
+                  $title = TicketTask::getTypeName($nb);
```

Comment:
`$nb` is useless (hardcoded to `0`), it should be dropped. And you first use the singular form. With `0`, you will get the plural form. Use `1` to get singular, and `Session::getPluralNumber()` to get plural.

Reply:
ok, corrected